diesel_cli/tests/support/mod.rs

```diff
@@ -1,3 +1,18 @@
+macro_rules! try_drop {
+    ($e:expr, $msg:expr) => { match $e {
+        Ok(x) => x,
+        Err(e) => {
+            use ::std::io::{Write, stderr};
+            if ::std::thread::panicking() {
```

Comment:
Since tests fail by panicking, that would be bad in any project (I'll bet `cargo test` sets `panic=unwind` even if the project is configured with `panic=abort`)